.github/workflows/continuous-integration.yml

```diff
@@ -112,41 +103,20 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - name: Use Node.js 14
-        uses: actions/setup-node@v2
-        with:
-          node-version: 14
+      - uses: actions/setup-node@v2
       - name: Install monorepo tools
         run: |
           npm ci --ignore-scripts
           npx lerna bootstrap --scope "@loopback/monorepo" --include-dependencies
       - name: Verify commit linting
         run: npx commitlint --from origin/master --to HEAD --verbose
 
-  ci-lint:
-    name: CI Lint
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
-      - uses: actions/setup-node@v2
-        with:
-          node-version: 14
-      - name: Install monorepo tools
-        run: npm ci --ignore-scripts
-      - name: Validate Renovate config
-        run: npx --no-install renovate-config-validator
-
```

Comment:
Without this validation step, Renovate will only be able to "react" to invalid configs **after** merge, which isn't ideal and can lead to a lot of trial-and-error and a messy Git history.

Reply:
Can we make it only run when the configuration file is modified?

Reply:
From  https://github.com/loopbackio/loopback-next/pull/8103:

```yaml
  ci-lint:
    name: CI Lint
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
        with:
          fetch-depth: 0
      - id: precheck
        name: Pre-check
        continue-on-error: true
        run: git diff --name-only origin/master @ | grep '.github/renovate.json5'
      - uses: actions/setup-node@v2
        if: steps.precheck.outcome == 'success'
        with:
          node-version: 16
      - name: Install monorepo tools
        if: steps.precheck.outcome == 'success'
        run: npm ci --ignore-scripts
      - name: Validate Renovate config
        if: steps.precheck.outcome == 'success'
        run: npx --no-install renovate-config-validator
```

Reply:
> From #8103:
> 
> ```yaml
>   ci-lint:
>     name: CI Lint
>     runs-on: ubuntu-latest
>     steps:
>       - uses: actions/checkout@v2
>         with:
>           fetch-depth: 0
>       - id: precheck
>         name: Pre-check
>         continue-on-error: true
>         run: git diff --name-only origin/master @ | grep '.github/renovate.json5'
>       - uses: actions/setup-node@v2
>         if: steps.precheck.outcome == 'success'
>         with:
>           node-version: 16
>       - name: Install monorepo tools
>         if: steps.precheck.outcome == 'success'
>         run: npm ci --ignore-scripts
>       - name: Validate Renovate config
>         if: steps.precheck.outcome == 'success'
>         run: npx --no-install renovate-config-validator
> ```

Better with the command `git diff --name-only origin/master @ | grep -Fx '.github/renovate.json5'`

Reply:
I have chosen to separate it in another workflow. Github has a limitation at the workflow level to filter by paths